.taskcluster.yml

```diff
@@ -1,160 +1,127 @@
+      # 1 represents anonymous/untrusted work (PRs)
+      then: 3
+      else: 1
+    repository_full_name:
+      $if: 'tasks_for == "github-pull-request"'
+      then: ${event.pull_request.base.repo.full_name}
+      else: ${event.repository.full_name}
+  in:
+    $let:
+      assume_scope_prefix: assume:repo:github.com/${repository_full_name}
+      default_task_definition:
+        taskId: ${decision_task_id}
+        taskGroupId: ${decision_task_id}  # Must be explicit because of Chain of Trust
+        schedulerId: taskcluster-github
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '2 hours'}
+        expires: {$fromNow: '1 year'}
+        provisionerId: aws-provisioner-v1
+        workerType: mobile-${trust_level}-decision
+        requires: all-completed
+        payload:
+          maxRunTime: 6000
+          image: mozillamobile/firefox-tv:2.3
+          command:
+            - /bin/bash
+            - --login
+            - -cx
+            # The rest of the command must be defined below
+          env:
+            TASK_ID: ${decision_task_id}
+            MOBILE_HEAD_REPOSITORY: ${repository}
+            MOBILE_HEAD_REV: ${head_rev}
+          features:
+            taskclusterProxy: true
+          artifacts:
+            public/task-graph.json:
+              type: file
+              path: /opt/firefox-tv/task-graph.json
+              expires: {$fromNow: '1 year'}
+            public/actions.json:
+              type: file
+              path: /opt/firefox-tv/actions.json
+              expires: {$fromNow: '1 year'}
+            public/parameters.yml:
+              type: file
+              path: /opt/firefox-tv/parameters.yml
+              expires: {$fromNow: '1 year'}
+        metadata:
+          name: 'Decision task'
+          description: ''
+          source: ${repository}/raw/${head_rev}/.taskcluster.yml
+    in:
+      - $if: 'tasks_for == "github-pull-request" && event["action"] in ["opened", "reopened", "edited", "synchronized"]'
+        then:
+          $mergeDeep:
+            - {$eval: 'default_task_definition'}
+            - scopes:
+                - ${assume_scope_prefix}:pull-request
+              payload:
+                command:
+                  - >-
+                    git fetch ${repository} ${event.pull_request.head.ref}
+                    && git config advice.detachedHead false
+                    && git checkout FETCH_HEAD
+                    && apt update
+                    && apt install -y python3-pip
+                    && pip3 install taskcluster
+                    && python3 tools/taskcluster/decision_task.py pull-request ${event.sender.login}@users.noreply.github.com ${event.pull_request.head.ref}
+              metadata:
+                owner: ${event.sender.login}@users.noreply.github.com
+      - $if: 'tasks_for == "github-push" && event["ref"] == "refs/heads/master"'
```

Comment:
So we don't do CI for our pushes to our release branches then? `releases/*`? That might be fine – we run CI on all PRs we uplift, run CI when building the release build, and generally don't check merges to `releases/*` anyway – but I just wanted to be explicit.

Reply:
That's correct :100: